misc/tools/atrium-samples-test/build.gradle.kts

```diff
@@ -3,7 +3,7 @@ plugins {
 }
 
 //TODO 1.2.0 include ...-kotlin-test again once we use IR for JS
-val samples = listOf("junit5", "spek", "kotest")//, "js-kotlin-test", "multiplatform-kotlin-test")
+val samples = listOf("junit5", "spek", "kotest", "testng")//, "js-kotlin-test", "multiplatform-kotlin-test")
```

Comment:
you also need to add it in settings.gradle.kts in this directory

Reply:
True, added. Thanks for the hint!